pkg/apply/solver/solver_test.go

```diff
@@ -38,10 +38,12 @@ func TestTaskQueueSolver_BuildTaskQueue(t *testing.T) {
 			objs:    []*unstructured.Unstructured{},
 			options: Options{},
 			expectedTasks: []taskrunner.Task{
+				&task.InvAddTask{},
```

Comment:
Should we consider these as any other task, so they will show up in the event stream? Or should we somehow consider these an implementation detail and hide them?

Reply:
Added inventory events.